test/integration/src/test/java/io/pravega/test/integration/StreamMetricsTest.java

```diff
@@ -255,6 +267,11 @@ public void testStreamsAndScopesBasicMetricsTests() throws Exception {
         assertEquals(1, (long) MetricRegistryUtils.getCounter(MetricsNames.UPDATE_SUBSCRIBER_FAILED, streamTags(failedRG, failedRG)).count());
     }
 
+    private long getTimerMillis(String timerName) {
+        val t = MetricRegistryUtils.getTimer(timerName);
```

Comment:
can you provide a meaningful name to the the `val t`?

Reply:
updated to timer